### DISCUSSION TITLE
New with CKAN

### DISCUSSION THREAD

**Discussion by kkounouni** [ckan/ckan#6761] [General]
Hello, I have the technical ownership of https://data.gov.cy which is in Drupal 7 with DKAN1. 

Now, we want to explore our options with CKAN. I have a testing environment with CKAN (the latest version). Do we need to have a CMS for the frontend in order to have a website like https://africaopendata.org/ and https://data.gov.sg?

Can we build workflow approvals?

How can we make dashboards?

Thanks in advance 
Katerina Kounouni

**Comment by categulario** [comment]
Hi @kkounouni !

We are probably in the same situation. I don't have answers to your questions but I think some of them can be addressed with theming, for instance the frontpage to some extent and I think a CMS is not necessary. (It is my task to setup a CKAN for a local government and I certainly have no interest in also maintaining a CMS)

**Comment by categulario** [comment]
And it turns out I also need to integrate a workflow for approvals, which I'm planing to solve with some theming tricks because it seems like there's no easy way to do it from an install of CKAN in the current version.
